export declare const catchError: (res: any) => Promise<any>;
